### DISCUSSION TITLE
[Enhancement]: remember last page open

### DISCUSSION THREAD

**Discussion by lance-374** [ciderapp/Cider#645] [Added Features]
### The feature you would like

Add the option to return to the last open tab in the sidebar to match iTunes and Apple Music apps. Also include the option to set a default page other than Home.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Amaru8** [comment]
**Hi @lance-374, I have some great news!** This feature is now implemented in the latest development version of **Cider**, and will be included in the next Microsoft Store update.
